frontend/src/api/ApiCaller.tsx

```diff
@@ -134,7 +134,7 @@ export class BackendAPICaller {
     static async getEnabledRobots(): Promise<Robot[]> {
         const path: string = 'robots'
         const result = await BackendAPICaller.GET<Robot[]>(path).catch(BackendAPICaller.handleError('GET', path))
-        return result.content.filter((robot) => robot.enabled)
```

Comment:
Note that the name of this function (getEnabledRobots) hasn't been changed, as this function will be rewritten to sort on deprecated instead in separate PR